meetings/2022-02-14.md

```diff
@@ -0,0 +1,112 @@
+* Use panel chat and repository. Queue in call to talk.
+
+
+### Participation and Code of Conduct
+* [Join the W3C Solid Community Group](https://www.w3.org/community/solid/join), [W3C Account Request](http://www.w3.org/accounts/request), [W3C Community Contributor License Agreement](https://www.w3.org/community/about/agreements/cla/)
+* [Solid Code of Conduct](https://github.com/solid/process/blob/master/code-of-conduct.md), [Positive Work Environment at W3C: Code of Ethics and Professional Conduct](https://github.com/solid/process/blob/master/code-of-conduct.md)
+* If this is your first time, welcome! please introduce yourself.
+
+
+### Scribes
+* Nicolas
+* Pavlik
+
+---
+
+## Agenda
+
+### Pull Requests
+
+* [created WebID Document section #79](https://github.com/solid/solid-oidc/pull/79)
+
+
+### Issues
+https://github.com/solid/solid-oidc/issues
+
+* [Authorization server behaviour for Client-Credentials Grant flow #75](https://github.com/solid/solid-oidc/issues/75) 
+* [Clarify implicitly trusted OIDC issuer #51](https://github.com/solid/solid-oidc/issues/51)
+* 
+
+## Minutes
+
+### Introductions
+
+* LD: Architect working with the Flemish government implementing Solid use cases.
+* AC: Note that in order to contribute to any of the solid panels, it's better to join the Working Group (see the Participation & CoC section)
+
+### Created WebID Document section
+
+[PR#79](https://github.com/solid/solid-oidc/pull/79)
+
+* EP: This is a first bite-size PR to the overall restructuration. The WebID document section has been added as a conformance class. Verification of the WebID validity has been added in some conformance classes, e.g. in OIDC issuer discovery for the RS conformance class. Implicit trust to same-domain OP has been removed as well (see dedicated section for more details). 
+* LD: It is mentioned that a WebID profile may trust multiple OP. What should we do there ? Could more metadata help making an informed decision.
+* EP: There is a taskforce emering for the WebID document, which is considering if a preferred OIDC issuer should be specified.
+* AC: This question has come up in multiple contexts. An interpretation to the presence of multiple OP is that the user may be presented with multiple options to log in (from the Client point of view). This means involving the user at this point would be a desirable pattern.
+* EP: Note that ACP also allows to restrict issuers when accessing some data. If the user is logged in multiple trusted OP, then it may have multiple ID token to select from.
+* NAS: Currently browser based clients wouldn't support being logged into multiple OP at once. We only have full window redirect, and it would require popup redirect. It could be supported on the server side but we haven't actively studied it. We'll have to check how it applies.
```

Comment:
```suggestion
* AC: This question has come up in multiple contexts. An interpretation of the presence of multiple OP is that the user may be presented with multiple options to log in (from the Client point of view). This means involving the user at this point would be a desirable pattern.
* EP: Note that ACP also allows to restrict issuers when accessing some data. If the user is logged in multiple trusted OP, then it may have multiple ID tokens to select from.
* NAS: Currently, browser based clients wouldn't support being logged into multiple OP at once. We only have full window redirect, and it would require popup redirect. It could be supported on the server side, but we haven't actively studied it. We'll have to check how it applies.
```